ci-operator/step-registry/ipi/conf/azure/sharednetwork/ipi-conf-azure-sharednetwork-commands.sh

```diff
@@ -5,20 +5,37 @@ set -o errexit
 set -o pipefail
 
 # TODO: move to image
-curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+curl -L https://github.com/mikefarah/yq/releases/download/v4.22.1/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
-PATCH=/tmp/install-config-sharednetwork.yaml.patch
 
-azure_region=$(/tmp/yq r "${CONFIG}" 'platform.azure.region')
+if [ -f "${SHARED_DIR}/customer_vnet_subnets.yaml" ]; then
+  vnet_name=$(cat ${SHARED_DIR}/customer_vnet_subnets.yaml | /tmp/yq '.platform.azure.virtualNetwork')
+  controlPlaneSubnet=$(cat ${SHARED_DIR}/customer_vnet_subnets.yaml | /tmp/yq '.platform.azure.controlPlaneSubnet')
+  computeSubnet=$(cat ${SHARED_DIR}/customer_vnet_subnets.yaml | /tmp/yq '.platform.azure.computeSubnet')
+  
+  PATCH="${SHARED_DIR}/install-config-qe-sharednetwork.yaml.patch"
 
-cat > "${PATCH}" << EOF
+  cat >> "${PATCH}" << EOF
+platform:
+  azure:
+    networkResourceGroupName: os4-common
```

Comment:
`os4-common`? I think it should be the provisioned resource group where vent is created.